packages/gateway/scripts/README.md

```diff
@@ -0,0 +1,27 @@
+# Gateway CLI
+
+# `denylist sync`
+
+Reads `wrangler.toml` to pick out the correct KV to write to based on the passed `--env` value.
+
+Requires a Cloudflare API token (as an environment variable) in order to write entries.
+
+Usage:
+
+```sh
+CF_API_TOKEN=<token> node scripts/cli.js denylist sync --env production
+```
+
+It reads from the local `denylist.json` as well as the [badbits denylist](https://badbits.dwebops.pub/denylist.json). Sources can be updated in `denylist.js`.
+
+# `denylist add`
+
+Add a CID (or CID + path) to the local deny list.
```

Comment:
Can we add instructions on how to do CID + path? I think an example would be good